manifests/l/LutzRoeder/Netron/5.7.0/LutzRoeder.Netron.locale.en-US.yaml

```
@@ -0,0 +1,21 @@
+PackageIdentifier: LutzRoeder.Netron
+PackageVersion: 5.7.0
+PackageName: Netron
+PackageLocale: en-US
+PackageUrl: https://github.com/lutzroeder/netron
+Publisher: Lutz Roeder
+PublisherUrl: https://github.com/lutzroeder/netron
+PublisherSupportUrl: https://github.com/lutzroeder/netron/issues
+Author: Lutz Roeder
+License: Copyright (c) Lutz Roeder
```

A copyright is not a license
```suggestion
License: MIT
LicenseUrl: https://github.com/lutzroeder/netron/blob/main/LICENSE
```